integration_tests/dataproc_test_case.py

```diff
@@ -94,7 +94,27 @@ def createCluster(self,
                       worker_accelerator=None,
                       optional_components=None,
                       machine_type="e2-standard-2",
-                      boot_disk_size="50GB"):
+                      boot_disk_size="50GB",
+                      startup_script=None):
+        createCluster(self, configuration, init_actions, metadata, scopes, properties, timeout_in_minutes,
+                beta, master_accelerator, worker_accelerator, optional_components, machine_type, machine_type,
+                boot_disk_size, startup_script)
+
+    def createCluster(self,
```

Comment:
This looks overloaded - can you please revert the additional function signature?

Reply:
its overloaded on purpose so that you can pass in different master from worker, I'll separate them out so has different function name. It seems to be confused by the machine-type parameter and I was only running hte mig test so didn't catch this

Reply:
fixed the cluster name issue and I get by the error you listed, but I don't have enough quota to actually run the tests al lthe way through

Reply:
Updated to add `master_machine_type` and `worker_machine_type` as optional parameters in the original function.